kinto/tests/support.py

```diff
@@ -51,6 +51,7 @@ def get_app_settings(self, additional_settings=None):
         settings['cliquet.project_docs'] = 'https://kinto.rtfd.org/'
         settings['multiauth.authorization_policy'] = (
             'kinto.tests.support.AllowAuthorizationPolicy')
+        settings['cliquet.userid_hmac_secret'] = "azerty"
```

Comment:
I would rather use a secret which explicitly says it's not secret (`this is not secret`)